alicloud/resource_alicloud_instance_test.go

```diff
@@ -1151,7 +1181,7 @@ resource "alicloud_instance" "foo" {
 	availability_zone = "${data.alicloud_zones.default.zones.0.id}"
 
 	instance_type = "${data.alicloud_instance_types.default.instance_types.0.id}"
-	internet_charge_type = "PayByBandwidth"
```

Comment:
I think, if provider is in China, there is no need to change internet_charge_type.

Reply:
The test is in fact executed in the current region, not just cn-beijing anymore. I have removed the misleading comment "# cn-beijing".